googleapis/google-cloud-node
# migrate code from googleapis/nodejs-appengine-admin

- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- feat!: initial stub of library
- feat: add samples
- fix: start at Node 12
- Revert "fix: start at Node 12"
- Revert "feat: add samples"
- feat: add initial samples and tests (#2)
- chore: release 0.2.0 (#12)
- build: fix repo metadata (#21)
- chore(deps): update dependency @types/sinon to v10 (#26)
- build: update package.json (#29)
- chore(deps): update dependency ts-loader to v9 (#28)
- build: update links for service-management, update package name for appengine-admin (#34)
- build: add generated-files bot config (#36)
- build: update to falsy behavior in protos (#38)
- chore: migrate to owl bot (#22)
- fix(deps): require google-gax v2.12.0 (#39)
- chore: new owl bot post processor docker image (#41)
- fix: use require() to load JSON protos (#42)
- chore: update gapic-generator-typescript to v1.3.2 (#43)
- chore: release 0.2.1 (#40)
- chore: release 1.0.0 (#47)
- chore(deps): update dependency sinon to v11 (#48)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#49)
- chore: release 1.0.1 (#50)
- fix: make request optional in all cases (#56)
- chore: release 1.0.2 (#57)
- fix(deps): google-gax v2.17.0 with mTLS (#62)
- chore: release 1.0.3 (#64)
- fix(deps): google-gax v2.17.1 (#65)
- chore: release 1.0.4 (#66)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#67)
- chore: release 1.0.5 (#68)
- chore(nodejs): update client ref docs link in metadata (#73)
- fix(deps): google-gax v2.24.1 (#76)
- chore: release 1.0.6 (#77)
- feat: turns on self-signed JWT feature flag (#78)
- chore: release 1.1.0 (#79)
- chore: disable renovate dependency dashboard (#1194) (#80)
- build(node): update deps used during postprocessing (#1243) (#87)
- chore(cloud-rad): delete api-extractor config (#88)
- chore: fix the wrong post processor image (#90)
- chore(deps): update dependency @types/node to v16 (#89)
- chore(deps): update dependency sinon to v12 (#91)
- docs(samples): add example tags to generated samples (#92)
- build: add generated samples to .eslintignore (#93)
- docs(node): support "stable"/"preview" release level (#1312) (#96)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#97)
- chore: add api_shortname and library_type to repo metadata (#95)
- test(nodejs): remove 15 add 16 (#1322) (#99)
- chore: add codeowners (#105)
- build: update copyright year to 2022 (#103)
- chore: update v2.12.0 gapic-generator-typescript (#106)
- chore(deps): update dependency sinon to v13 (#108)
- docs(nodejs): version support policy edits (#1346) (#110)
- chore: update repo-metadata.json (#102)
- chore: cleanup unused proto imports (#112)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#113)
- chore: update v2.14.2 gapic-generator-typescript (#118)
- feat: promote to stable (#119)
- chore(main): release 1.2.0 (#120)
- build(node): update client library version in samples metadata (#1356) (#127)
- chore(deps): update dependency @types/mocha to v9 (#128)
- feat: add Application.service_account (#130)
- chore(deps): update dependency sinon to v14 (#133)
- build!: update library to use Node 12 (#134)
- chore(deps): update dependency jsdoc-fresh to v2 (#138)
- chore(deps): update dependency jsdoc-region-tag to v2 (#139)
- chore(main): release 2.0.0 (#136)
- feat: support regapic LRO (#140)
- chore(main): release 2.1.0 (#141)
- chore(deps): update dependency linkinator to v4 (#146)
- chore: remove unused proto imports (#149)
- fix: better support for fallback mode (#150)
- fix: change import long to require (#152)
- chore(main): release 2.1.1 (#153)
- fix: do not import the whole google-gax from proto JS (#1553) (#155)
- fix: allow passing gax instance to client constructor (#156)
- fix: preserve default values in x-goog-request-params header (#158)
- test: use fully qualified request type name in tests (#160)
- fix(deps): use google-gax v3.5.2 (#164)
- chore(deps): update dependency jsdoc to v4 (#166)
- fix: regenerated protos JS and TS definitions (#167)
- chore(deps): update dependency @types/node to v18 (#162)
- chore(main): release 2.1.2 (#157)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
